themes/buildpacks/assets/scss/components/_docs.scss

```diff
@@ -70,7 +70,17 @@
       }
       a {
         text-decoration: none;
-        border-bottom: 1px solid rgba($blue-dark, 0.25);
+        margin-left: -1.5rem;
+        .h-anchor {
+          font-size: 1.25rem;
```

Comment:
Opinion: The icon looks a little large in certain areas. It should probably scale based on header size. Maybe we want `em` here instead of `rem`.

Ref: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#Relative_length_units

Reply:
sounds good. I used `rem` originally so I could use a set `margin-left` for the header, but I can play around with it